frontend/layouts/admin.vue

```diff
@@ -43,7 +43,7 @@ const { $globals } = useNuxtApp();
 
 const sidebar = ref<boolean>(false);
 onMounted(() => {
-  sidebar.value = !display.md.value;
+  sidebar.value = !display.lgAndUp.value;
```

Comment:
I think this should be

```suggestion
  sidebar.value = display.lgAndUp.value;
```